JavaScript/JavaScriptSDK/Initialization.ts

```diff
@@ -130,6 +130,7 @@ module Microsoft.ApplicationInsights {
             config.verboseLogging = Util.stringToBoolOrDefault(config.verboseLogging);
             config.emitLineDelimitedJson = Util.stringToBoolOrDefault(config.emitLineDelimitedJson);
             config.diagnosticLogInterval = config.diagnosticLogInterval || 10000;
+            config.samplingPercentage = config.samplingPercentage || 100;
```

Comment:
Alex is right, 0 will evaluate to false and 100 will be used, totally unintentionally. Vitaly, I'll sync it with server side logic.